app/controllers/publish/courses/interview_process_controller.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Publish
+  module Courses
+    class InterviewProcessController < PublishController
+      include CopyCourseContent
+      before_action :authorise_with_pundit
+
+      def edit
+        @interview_process_form = CourseInterviewProcessForm.new(course_enrichment)
+        @copied_fields = copy_content_check(::Courses::Copy::INTERVIEW_PROCESS_FIELDS)
+
+        @copied_fields_values = copied_fields_values if @copied_fields.present?
+
+        @interview_process_form.valid? if show_errors_on_publish?
+      end
+
+      def update
+        @interview_process_form = CourseInterviewProcessForm.new(course_enrichment, params: interview_process_params)
+        @interview_process_form = CourseInterviewProcessForm.new(course_enrichment, params: interview_process_params)
+
+        if @interview_process_form.save!
+          course_updated_message I18n.t('publish.providers.interview_process.edit.interview_process_success')
+
+          redirect_to redirect_path
+        else
+          render :edit
+        end
+      end
+
+      private
+
+      def authorise_with_pundit
+        authorize provider
```

Comment:
TBH, I haven't really looked into this at all, I was just using the same policy that was used for all the controllers in this section. I'll have a look now -- but if it isn't entirely clear, or if it looks like it needs to be updated in a few places, I'll create a separate ticket for it.

Reply:
I agree, this should be course instead of provider, so I've updated it in this controller and made a [ticket](https://trello.com/c/LHEno2wU) to review the other controllers and update where required.